db/contacts.json

```diff
@@ -0,0 +1,143 @@
+{
+	"contacts": [{
+			"id": 1,
+			"firstName": "Gib",
+			"lastName": "Bowden",
+			"phone": "615-944-4766",
+			"email": "gibbowden@gmail.com",
+			"address": {
+				"street 1": "800 Woodland Street",
```

Comment:
You don't want nested objects with Firebase. You can either add them as single lines or create a new collection with of addresses with foreign keys on them. (I'd recommend going with the first option for this project).